spec/pb_kits/playbook/props_spec.rb

```diff
@@ -96,4 +96,23 @@
       end
     end
   end
+
+  describe "required props" do
+    subject do
+      Class.new do
+        include Playbook::Props
+
+        prop :required_prop, required: true
+      end
+    end
+
+    it "raises error when not given a value" do
+      expect { subject.new({}) }.to raise_error(Playbook::Props::Error)
+    end
+
+    it "does not raise error when given a value", :aggregate_failures do
+      expect { subject.new(required_prop: "value") }.to_not raise_error(Playbook::Props::Error)
```

Comment:
This first expectation can be removed as the next line won't blow up and shall pass